src/rust/engine/src/externs.rs

```diff
@@ -97,6 +97,10 @@ pub fn store_i64(val: i64) -> Value {
   with_externs(|e| (e.store_i64)(e.context, val).into())
 }
 
+pub fn store_bool(val: bool) -> Value {
+  with_externs(|e| (e.store_bool)(e.context, val).into())
+}
```

Comment:
Here we're assuming that the rust `bool` is necessarily going to correspond to the C/C++ `_Bool` type generated in `native_engine.c` (from `stdbool.h`) -- I assume this is safe, but why? I'm seeing conversions to `i32` in the [`bool` docs](https://doc.rust-lang.org/std/primitive.bool.html), which differs from e.g. this excerpt in `native_engine.c` (admittedly for a platform we don't support):
```
# if _MSC_VER < 1800   /* MSVC < 2013 */
#  ifndef __cplusplus
    typedef unsigned char _Bool;
#  endif
```

Reply:
And yes, I saw the signature of `extern_equals`/etc, so I wasn't concerned, was more asking if this works because everything would break if we couldn't assume it or if it works because we're doing something specific. Thanks!